lib/bolt/plugin/terraform.rb

```diff
@@ -21,7 +21,26 @@ def warn_missing_property(name, property)
         @logger.warn("Could not find property #{property} of terraform resource #{name}")
       end
 
+      # Make sure no unexpected keys are in the inventory config and
+      # that required keys are present
+      def validate_options(opts)
```

Comment:
Could also use `Set` logic here to check keys instead of iterating over lists. We do that for yaml validation. https://github.com/puppetlabs/bolt/blob/c049c3609ac2d3af88b9f772cdfac93d4950485d/lib/bolt/pal/yaml_plan.rb#L34-L38